pkg/networking/cilium/client.go

```diff
@@ -11,10 +11,10 @@ import (
 )
 
 const (
-	ciliumDaemonSetName           = "cilium"
-	ciliumPreflightDaemonSetName  = "cilium-pre-flight-check"
-	ciliumDeploymentName          = "cilium-operator"
-	ciliumPreflightDeploymentName = "cilium-pre-flight-check"
+	DaemonSetName           = "cilium"
```

Comment:
just out of curiosity, why did we remove the cilium prefix?

Reply:
To use "cilium.DaemonSetname" instead of "cilium.ciliumDaemonSetName" outside the package. The package name already serves as namespace for the constants.